app/views/versions/index.html.erb

```diff
@@ -10,4 +10,9 @@
       <%= render @versions %>
     </ul>
   </div>
+  <% if @versions.any?(&:rely_on_built_at?) %>
+    <div class="imported-version-notice t-list__items">
+      <p><%= t('.imported_gem_version_notice', import_date: nice_date_for(Version::RUBYGEMS_IMPORT_DATE)) %></p>
```

Comment:
I am not sure I understand why this is best place to add it. The gem show page will have * mark for version but no explanation but if they click on "all versions" then they can see the explanation.
Would adding tooltip next to version which shows this on hover work better?

Reply:
To be honest I just forgot about the fact that we show 5 most recent versions on the `show` page and was mainly focused on the versions `index` page.
Having a tooltip makes sense, let me quickly add that
I think that having a tooltip should be sufficient, but if anyone thinks that we may want to have both the tooltip and a separate section at the bottom feel free to speak up! Thanks